docs/advanced.md

```diff
@@ -67,7 +67,9 @@ The library uses 4 different logging levels:
 Check out this [example out for a live demonstration](../examples/custom-logging).
 
 ## Change the generated indentation type and size
-TODO 
+In some scenarios, depending on how you stitch them together, you might need to change the indentation type or size both of these cases are fully supported.
```

Comment:
I'm not a expert of English but maybe that `and` sounds better:

```suggestion
In some scenarios, depending on how you stitch them together, you might need to change the indentation type or size and both of these cases are fully supported.
```

@alequetzalli What do you think? 🤔